drafts/configure-logs-JSON-format-190010.adoc

```diff
@@ -61,29 +62,32 @@ Or take a look at our link:/downloads/[Downloads page].
 image::/img/blog/blog_btn_stack.svg[Ask a question on Stack Overflow, align="center"]
 
 [#jlog]
-== Configurable JSON log field names
+== Configurable JSON log field names (logging)
```

Comment:
Don't think you need the "(logging)" bit of the heading - I don't think it adds meaning. The bits in parentheses are usually the name of the feature (not sure if logging is specific to a feature or just part of the OL kernel). Don't think it's a problem to be inconsistent and not have it for this heading.